notifier.rst

```diff
@@ -516,12 +517,11 @@ very high and the recipient has a phone number::
             $this->price = $price;
         }
 
-        public function getChannels(Recipient $recipient)
+        public function getChannels(RecipientInterface $recipient)
```

Comment:
Please add missing use statements (same for `SmsRecipientInterface`)

Reply:
done